### DISCUSSION TITLE
[QUESTION] How to develop?

### DISCUSSION THREAD

**Discussion by nikkwong** [GrapesJS/grapesjs#4199] [Q&A]
I'm trying to just get the dev server using HMR to show changes that I'm making to the editor. However, when i run `npm run start` and make changes to the code in `src`, the changes don't appear in the editor (changing or rearranging buttons, etc). What's the trick to get this working? Thanks.

I can see that the index.html file is referencing `grape.min.js` in `dist`, but `dist` is not getting rebuilt on changes.

**Comment by ronaldohoch** [comment]
Sometimes, when i'm developing a plugin, i need to force a reload, did you tryed it?

**Comment by artf** [comment]
@nikkwong did you download the latest `dev` branch? Can you give more details about your environment? The dev server should reload your page on file changes
